packages/components/src/passengerCard/PassengerCard.js

```diff
@@ -64,13 +72,15 @@ class PassengerCard extends React.Component<Props> {
       passengerCount,
       bags,
       actionIconName,
+      secondaryActionIconName,
       onActionPress,
+      onSecondaryActionPress,
```

Comment:
Are we using this card for something else? Can we just rename it `onEditPress` and `onDeletePress`?

Reply:
All done!

Reply:
Thanks!